models/credmark/compose/each.py

```diff
@@ -0,0 +1,41 @@
+
+
+
+
+from typing import List, Union
+from credmark.cmf.model import Model, describe
+from credmark.cmf.model.errors import ModelRunError
+from credmark.dto import DTO, IterableListGenericDTO
+
+class ModelComposeEachDTO(DTO):
+    model_slug:str
+    model_input_slug: str
+    model_input_input: DTO
+
+class ModelComposeEachOutputRowDTO(DTO):
+    input:Union[dict,DTO]
+    output:Union[dict,DTO]
+    
+class ModelComposeEachOutputDTO(IterableListGenericDTO):
+    _iterator:str = "results"
+    results: List[ModelComposeEachOutputRowDTO]
+
+@describe(
+    slug="compose.each",
+    version="1.0",
+    input=ModelComposeEachDTO,
+    output=ModelComposeEachOutputDTO)
+class ComposeEach(Model):
+    def run(self, input: ModelComposeEachDTO) -> ModelComposeEachOutputDTO:
+        list_model = self.context.run_model(
+            slug=input.model_input_slug,
```

Comment:
Maybe this should be "fetcher_slug"? Was that the name we decided on for models that do Data Fetch?

Reply:
Better to nest: `input.fetch.slug` `input.fetch.input`

```
input = {
   "fetch": {
      "slug": "foo",
      "input": {
          ....
      }
   }
   "process": {
      "slug": "bar",
   }
}
```